src/DeploymentManager/DeploymentManager/Commands/SetArtifactSource.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.DeploymentManager.Commands
+{
+    using System.Management.Automation;
+
+    using Microsoft.Azure.Commands.DeploymentManager.Models;
+
+    [Cmdlet(
+        VerbsCommon.Set,
+        ResourceManager.Common.AzureRMConstants.AzurePrefix + "DeploymentManagerArtifactSource",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = DeploymentManagerBaseCmdlet.InputObjectParamSetName),
+     OutputType(typeof(PSArtifactSource))]
+    public class SetArtifactSource : DeploymentManagerBaseCmdlet
```

Comment:
@deveshguha we should add the interactive and resource id parameter sets to this cmdlet, and add parameters for the properties that can be updated on the object

Reply:
This is not an update to our cmdlets, but merely a move from AzureRM to Az format, so do not want to add more funtionality at this time, for this preview version. Will update these in the subsequent versions.

Reply:
@deveshguha okay, this will need to be done before the module can be considered stable, but it's fine for the preview release now